src/devices/machine/i82586.h

```diff
@@ -9,28 +9,254 @@
 #define MCFG_I82586_IRQ_CB(_out_irq) \
 	devcb = &i82586_base_device::static_set_out_irq_callback(*device, DEVCB_##_out_irq);
 
+#define I82586_SCP_ADDRESS 0xfffff4
+
+// FIXME: datasheet says this field should be "all 1's", but InterPro sets only lower 16 bits (in linear mode)
+#define I82586_TBD_EMPTY   0xffff
+#define I82586_RBD_EMPTY   0xffff
+
+#define I82586_CFG_SIZE 11
+#define I82596_CFG_SIZE 14
+
+#define I82586_DUMP_SIZE 170
+#define I82596_DUMP_SIZE 304
+
+#define FCS_RESIDUE 0xdebb20e3
```

Comment:
Do these constants really need to be at global scope, and do they really have to be `#define` macros?  These things tend to cause annoying compile errors when you get collisions.